todos-app/frontend/src/components/TaskCreateModal.vue

```diff
@@ -32,6 +32,17 @@
 
             <label class="form-label mt-4">Due Date</label>
             <input type="datetime-local" class="form-control" v-model="task.due_datetime">
+
+            <label class="form-label mt-4">Tags</label>
+            <div class="d-flex flex-wrap">
+              <span v-for="(label, i) in task.labels" :key="i" class="badge rounded-pill badge-outline-secondary m-1">#{{label}}<i type="button" class="bi bi-x" @click="removeLabel(label)"></i></span>
+            </div>
+            <div class="input-group">
+              <input name="label" type="text" class="form-control" placeholder="Max 15 characters without space" v-model="labelInput">
+              <button type="button" class="btn btn-outline-secondary input-group-text" @click="addLabel">
+                <i class="bi bi-plus-circle"></i>
+              </button>
+            </div>
```

Comment:
![image](https://user-images.githubusercontent.com/10822260/149431158-7cefda9f-eae0-42b5-b866-b3f5ecb8c2c7.png)

Wow. You are good at creating frontend UI. How long does it take you to come out with this part?

Reply:
I got the idea from Vuetify's chip (https://vuetifyjs.com/en/components/chips/), but I couldn't use Vuetify in this project because it's not compatible with Vue 3 yet. I wanted to try out Vue 3 so I had to go with vanilla Bootstrap this time 🥲
I played around with css a bit to copy the looks 😁